src/utils/_opaciy.scss

```diff
@@ -0,0 +1,14 @@
+// --- Opacity --- //
+
+// alpha parameter for rgba Colors
+// @include bgcolor-opacity($c-black);
+
+$opacity-value: (15, 25, 50, 75, 100) !default;
+
+@mixin bgcolor-opacity ($color) {
```

Comment:
We already have this function defined: https://github.com/SUI-Components/sui-theme/blob/master/src/utils/_colors.scss#L15, does the new one something different?, and, should we follow different solutions to apply transparency to colors?

Reply:
Ok, I see, this only applies to background opacities, nothing more to say 😄

Reply:
yes, I was just going to write